plugins/org.eclipse.fordiac.ide.debug.ui/src/org/eclipse/fordiac/ide/debug/ui/fb/FBLaunchConfigurationTab.java

```diff
@@ -98,12 +111,93 @@ protected Composite createEventComponent(final Composite parent) {
 		return group;
 	}
 
+	protected Composite createDebugTimeComponent(final Composite parent) {
+		Label debugTimeLabel;
+		final Group group = new Group(parent, SWT.BORDER);
+		GridLayoutFactory.swtDefaults().applyTo(group);
+		group.setText("Debug Time"); //$NON-NLS-1$
+
+		final Composite comp = new Composite(group, SWT.NONE);
+		GridLayoutFactory.swtDefaults().numColumns(3).applyTo(comp);
+		GridDataFactory.fillDefaults().grab(true, false).applyTo(comp);
+
+		systemTimeRadio = new Button(comp, SWT.RADIO);
+		systemTimeRadio.setText("System"); //$NON-NLS-1$
+		systemTimeRadio.addSelectionListener(new SelectionAdapter() {
+			@Override
+			public void widgetSelected(final SelectionEvent e) {
+				updateLaunchConfigurationDialog();
+				setCheckbox(systemTimeRadio.getText());
+			}
+		});
+		GridDataFactory.fillDefaults().applyTo(systemTimeRadio);
+		incrementTimeRadio = new Button(comp, SWT.RADIO);
+		incrementTimeRadio.setText("Increment"); //$NON-NLS-1$
+		incrementTimeRadio.addSelectionListener(new SelectionAdapter() {
+			@Override
+			public void widgetSelected(final SelectionEvent e) {
+				updateLaunchConfigurationDialog();
+				setCheckbox(incrementTimeRadio.getText());
+			}
+		});
+		GridDataFactory.fillDefaults().applyTo(incrementTimeRadio);
+		manualTimeRadio = new Button(comp, SWT.RADIO);
+		manualTimeRadio.setText("Manual"); //$NON-NLS-1$
+		manualTimeRadio.addSelectionListener(new SelectionAdapter() {
+			@Override
+			public void widgetSelected(final SelectionEvent e) {
+				updateLaunchConfigurationDialog();
+				setCheckbox(manualTimeRadio.getText());
+			}
+		});
+		GridDataFactory.fillDefaults().applyTo(manualTimeRadio);
+
+		final Composite c = new Composite(group, SWT.NONE);
+		GridLayoutFactory.swtDefaults().numColumns(3).applyTo(c);
+		GridDataFactory.fillDefaults().grab(true, false).applyTo(c);
+
+		debugTimeText = new Text(c, SWT.BORDER);
+		debugTimeText.setText("0"); //$NON-NLS-1$
+		debugTimeText.setLayoutData(new GridData(SWT.FILL, SWT.FILL, true, false, 1, 1));
+		debugTimeText.addSegmentListener(e -> updateLaunchConfigurationDialog());
+		debugTimeLabel = new Label(c, SWT.NONE);
+		debugTimeLabel.setText("ms"); //$NON-NLS-1$
+		debugTimeLabel.setLayoutData(new GridData(SWT.LEFT, SWT.CENTER, true, false, 1, 1));
+
+		return group;
+	}
+
+	protected static void setCheckbox(final String text) {
+		// only the most recent selected box is checked
```

Comment:
I believe this is not necessary, since all RADIO buttons inside a container should update their selected state to be mutually-exclusive automatically.

Reply:
yes. fixed.